face/UnifiedFaceCollection/python/unified_face_collection.py

```diff
@@ -175,223 +188,140 @@ def remove_person(self, person_identifier, delete_faces=False):
-            if person_group_status['status'] == 'succeeded':
-                break
-            elif person_group_status['status'] == 'failed':
-                print("Large Person Group training failed")
-                return False
-            time.sleep(1)
+        poller_person_group = self.face_admin_client.large_person_group.begin_train(
+            large_person_group_id=self.large_person_group_id,
+            polling_interval=5,
+        )
+        poller_person_group.wait()
+        print(f"The person group {self.large_person_group_id} is trained successfully.")
         
         return True
 
     def find_face(self, image_path, search_type='face'):
         # Detect face in the image
-        detected_faces = detect_faces(self.subscription_key, self.endpoint, image_path, self.injection_header)
+        detected_faces = self.detect_faces(image_path)
         face_ids = [face['faceId'] for face in detected_faces]
         if not face_ids:
             return []
 
         normalized_results = []
         if search_type == 'person':
             # Find similar persons in the Large Person Group
-            identify_url = f"{self.face_api_url}/identify"
-            identify_data = {
-                'faceIds': [face_ids[0]],
-                'largePersonGroupId': self.large_person_group_id,
-                'maxNumOfCandidatesReturned': 10,
-            }
-            identify_response = requests.post(identify_url, headers=self.headers, json=identify_data)
-            if identify_response.status_code == 200:
-                results = identify_response.json()
-                for result in results:
-                    for candidate in result.get('candidates', []):
-                        normalized_results.append({
-                            'personId': candidate['personId'],
-                            'confidence': candidate['confidence']
-                        })
-                return normalized_results
-            else:
-                return []
-        else:
-            # Find similar faces in the Large Face List
-            similar_faces_url = f"{self.face_api_url}/findsimilars"
-            similar_faces_data = {
-                'faceId': face_ids[0],
-                'largeFaceListId': self.large_face_list_id,
-                'maxNumOfCandidatesReturned': 10,
-            }
-            similar_faces_response = requests.post(similar_faces_url, headers=self.headers, json=similar_faces_data)
-            if similar_faces_response.status_code == 200:
-                results = similar_faces_response.json()
-                for result in results:
+            identify_results = self.face_client.identify_from_large_person_group(
+                face_ids=[face_ids[0]],
+                large_person_group_id=self.large_person_group_id
+            )
+
+            for result in identify_results:
+                for candidate in result.get('candidates', []):
```

Comment:
```suggestion
                for candidate in result.candidates:
```

See https://learn.microsoft.com/en-us/python/api/azure-ai-vision-face/azure.ai.vision.face.models.faceidentificationresult?view=azure-python-preview#azure-ai-vision-face-models-faceidentificationresult-candidates.